server/controller/urlController.js

```diff
@@ -1,7 +1,9 @@
-const urlModel = require("../model/urlSchema");
+const { urlModel, userModel } = require("../model/urlSchema");
```

Comment:
**userSchema** should be in separate file.
Create a file named **userModel** inside models folder and then add **userSchema**. 
Finally update the path here.